src/helpers/document.js

```diff
@@ -77,23 +78,29 @@ const init = () => {
         }
 
         //Backspace was pressed
-        if (e.keyCode === 8) {
-            dirFilter = dirFilter.slice(0, -1);
-            $filter.innerHTML = dirFilter;
-            printSubdirectories();
-        }
+        if (e.keyCode === 8)
+            store.setState({
+                directoryFilter: directoryFilter.slice(0, -1)
+            });
 
         // Any a-z letter was pressed
-        if (/^[A-Z]$/i.test(e.key)) {
-            dirFilter += e.key;
-            $filter.innerHTML = dirFilter;
-            printSubdirectories();
-        }
+        if (/^[A-Z]$/i.test(e.key))
```

Comment:
What is the project is called something like `git-easy`, I wouldn't be able to write the `-` on the filter. If you think this is something we should do, please create another issue for it.